packages/block-editor/src/components/link-control/index.js

```diff
@@ -404,20 +405,24 @@ function LinkControl( {
 								createSuggestionButtonText
 							}
 							hideLabelFromVision={ ! showTextControl }
+							suffix={
+								showActions ? undefined : (
+									<InputControlSuffixWrapper variant="control">
+										<Button
+											onClick={
+												isDisabled ? noop : handleSubmit
+											}
+											label={ __( 'Submit' ) }
+											icon={ keyboardReturn }
+											className="block-editor-link-control__search-submit"
+											aria-disabled={ isDisabled }
+											size="small"
+										/>
+									</InputControlSuffixWrapper>
+								)
+							}
+							props
```

Comment:
I just noticed this `props` here which looks like a typo — should we remove it? Was it meant to be something like `{...props} ? @mirka

Reply:
Yes, I think that this is a typo; props were never there in the first place.

Reply:
Cleanup in #65650